### ISSUE TITLE
class:TaxonRelationshipAssertion

### ISSUE THREAD

**Issue by nielsklazenga** [tdwg/tcs2#43]
### TaxonRelationshipAssertion (property)

|    |    |
|----|----|
| Label | Taxon Relationship Assertion |
| Definition | Assertion of a relationship between two taxa |
| Usage notes |  |
| Comments |  |
| Required |  |
| Repeatable |  |
| Constraints |  |

**Comment by nielsklazenga** [comment]
We had a long discussion about the label for this class in https://github.com/tdwg/tnc/issues/48, but I do not think the last thing has been said yet.

At the moment, `TaxonRelationship` mainly seems too broad to me, as we want it to contain only the RCC-5-like relationships, so I think it needs to be qualified somehow.

**Comment by camwebb** [comment]
If we changed it to `TaxonConceptRelationship`, then, for me at least, it more naturally excludes many of the other kinds of relationship originally in TCS (child, anamorph, parent...). I'd rather not use RCC5 in the name, since RCC5 is a specific (spatial) definition of general set relationships. Using the characters `Set` somewhere in the name would work for me too.

**Comment by deepreef** [comment]
I know it's more letters, but I would prefer `TaxonCircumscriptionRelationship`.  The word "Concept" is so overloaded in taxonomy, with different meanings and interpretations and assumptions, etc.

**Comment by nielsklazenga** [comment]
I am wondering if we can come up with a term that qualifies the type of relationship, rather than the things the relationship is between.

**Comment by deepreef** [comment]
> I am wondering if we can come up with a term that qualifies the type of relationship, rather than the things the relationship is between.

I'm not sure I follow.  Do you mean something like `SetRelationship` or something?  Are you just trying to come up with a replacement to `TaxonRelationshipAssertion` of TCS 1?

**Comment by nielsklazenga** [comment]
This is the `TaxonRelationshipAssertion`/`TaxonRelationship` element of TCS 1. We cut back the relationship types that will be covered in this object to only the set relationships, so it would be good to reflect that in the label. I am thinking especially about separating them from the hierarchical relationships. So yes, something like 'SetRelationship', only I do not like that particular term. But it is good to get the options out there, so we can choose the best one when it comes to it.

I am not thinking about removing the `Taxon` bit, but the objects the relationship is between has not changed, so I do not think we have to change that bit. It is the `Relationship` bit that has changed, as that will be much narrower than it is in TCS 1.

**Comment by jliljeblad** [comment]
Maybe `TaxonOverlap` is more explicit on what kind of relationship this is about. In the same way, maybe the `intersects `could be replaced by simply `overlapsAtLeastSome`. Not pretty though, I agree.

**Comment by nielsklazenga** [comment]
I am sort of with @deepreef. This is a comparison of circumscriptions. Only thing is that this introduces a new term and I would not want to define `circumscription`, especially since I think that in `specimenCircumscription` and `characterCircumscription` it is used in a (slightly) different meaning than it would here. You would get discussions about whether a characterCircumscription can be congruent with a specimenCircumscription, or whether a circumscription based on morphological characters is disjoint with one based on molecular characters (or that a circumscription based on one character is disjoint with one based on another character in general). There might be an alternative for 'circumscription' that means approximately the same thing and does not have the same problems.

**Comment by nielsklazenga** [comment]
I think @jliljeblad is onto something as well. Region Connection Calculus is about connectedness. This does not make much sense for taxa (and also RCC-5 is not about connectedness anymore), so we could make it about intersection. So, maybe not the exact word 'overlap', but it is in the right direction. Maybe 'TaxonIntersection'? Then we can leave the controlled terms as they are.

I still would like something in between 'Taxon' and 'Intersection', like 'Circumscription' (but again, maybe not that exact word, unless we define it) and I think I would like the 'Assertion' back at the end.

Just throwing this out there in the hope the perfect term will reveal itself at some point.

**Comment by nielsklazenga** [comment]
From a taxonomic perspective, it might also make sense to make it about 'congruence'. I think I might like 'TaxonConceptCongruence'. If taxon concepts are not `congruent`, there is some sort of conflict and the other terms `has proper part`, `is proper part of` and `partially overlaps` describe the nature of the conflict. `intersects` indicates uncertainty (it could be congruent, or it could not be) and `is disjoint with` is like 'not applicable'.

**Comment by ghwhitbread** [comment]
I will suggest TaxonAlignment: Given that these relationships are about comparing circumscriptions, which, in all but instances of congruence, are about the relative instances of taxa (plural) in different taxonomic treatments. An unambiguous term, previously used by @nfranz.

**Comment by nielsklazenga** [comment]
I like 'Alignment' too, as it implies an opinion, so we do not have to consider adding 'Assertion' to the term. I think I'd like `TaxonConceptAlignment` better than `TaxonAlignment`.

**Comment by deepreef** [comment]
I also like 'Alignment', but I will again caution against using "Concept" in any of these terms.  That word just carries too much baggage (i.e., it means too many different things to different people).  Although it's more cumbersome, I think "Circumscription" is the word we want to stick with.  If `TaxonCircumscriptionAlignment` is too many characters, then I would go with `TaxonAlignment` over `TaxonConceptAlignment`.

**Comment by deepreef** [comment]
Just to be clear, I don't have any problem with the number of characters, but I really, strongly think we should avoid the word "Concept" as much as possible in any of these terms.

**Comment by nielsklazenga** [comment]
It is not just 'Concept', but 'TaxonConcept', which we define. I do not think the meaning is ambiguous, just that the word 'concept' might be used incorrectly, but taxonomists have been talking about 'taxon concepts' long before TCS, so I think we should stick with it. The subject and object of these alignments are `TaxonConcept`s, so I think it might be more confusing if we do not use it in the label.

'Circumscription' on the other hand is not defined and is used in TCS 1 in a different sense than is proposed here.

I am fine with `TaxonAlignment` though.

**Comment by deepreef** [comment]
`Taxon Concept` = circumscribed set of organisms
`Taxon Concept` = particular treatment, including both synonymy and classification
`Taxon Concept` = "Species Concept" (BSC, PSC, etc.)
`Taxon Concept` = concept of what a "taxon" is

The word "circumscription" has been defined in biodiversity informatics literature and is often used in TDWG meetings and such, and has only one meaning* (the first one in the list above, which is what we want).  I've lost countless minutes/hours at informatics discussions (Including the formulation of TCS 1) because people who thought they disagreed with each other turned out to be operating under different assumed meanings of "Taxon Concept".  The only term that leads to even more confusion, misunderstanding, etc. in this space is "Taxon Name" -- but that's a rant for another issue.

Often it's best to leverage a term that people are familiar with, to facilitate a shared understanding of the meaning.  But sometimes a term has multiple meanings within a community, leading to incorrect presumptions of its meaning.

*TCS 1 includes the *qualified* terms "SpecimenCircumscription" and "CharacterCircumscription".  The unqualified term "circumscription" has been consistently used more broadly (and in the documentation of TCS 1) in the sense of "complete set of organisms comprising this taxon".

Maybe the easiest path forward is just to go with `TaxonAlignment` -- which several of us seem happy with.  Does anyone object to that term?

**Comment by jgerbracht** [comment]
Even though I really want to use TaxonConcept, and still do when conversing with folks.  I also have spent hours in various conversations only to realize part way through that I'm speaking of a stable, circumscribed set of organisms and the other person is thinking of the usages of a taxonomic name, which is variable depending on the authority.  The only issue I have with TaxonCircumscription is that it could also mean the specific circumscription text and not the taxon or set of organisms being described.
   I can go either way as long as we very clearly define what we mean.
    Jeff
--
Jeff Gerbracht
Lead Application Developer
Birds of the World
Cornell Lab of Ornithology
607-254-2117

________________________________

**Comment by nielsklazenga** [comment]
> `Taxon Concept` = circumscribed set of organisms
`Taxon Concept` = particular treatment, including both synonymy and classification
`Taxon Concept` = "Species Concept" (BSC, PSC, etc.)
`Taxon Concept` = concept of what a "taxon" is


I would say a 'circumscribed set of organisms' is a `Taxon`. The treatment is not the same thing as the `Taxon Concept` in it, so the second and fourth are the same. Just because a species is a taxon does not mean that a taxon concept is the same as a species concept. And again, we are defining `TaxonConcept`, so possible different meanings are irrelevant.

Defining 'Circumscription' as the set of organisms it is a circumscription for is circular to me. 

From @jgerbracht 's post:
> The only issue I have with TaxonCircumscription is that it could also mean **the specific circumscription text** and not the taxon or set of organisms being described.

That is what a circumscription is, certainly in TCS, although not necessarily text, but a set of characters or representative specimens. Giving it the extra/different meaning of a shared idea of what the delimitation of a group of organisms is makes that `TaxonCircumscription` has all the problems `TaxonConcept` is purported to have.

**Comment by nielsklazenga** [comment]
The `Taxon` is the set of organisms, which is very much in accordance with the definition in Darwin Core, and since it is the sets of organisms we are trying to align – and also because it seems to be something we can agree on – it makes sense to go with `TaxonAlignment` (so @ghwhitbread had it right from the start).

**Comment by ghwhitbread** [comment]
I was also considering 'TaxonomicAlignment'.

**Comment by nielsklazenga** [comment]
Maybe you should quit while you are ahead :grin:. I have a problem with `taxonomic`, because it is the adjective for `taxonomy`, not `taxon`, so it makes the label really vague. Even within our broader discipline (outside systematics) it can be used for vegetation communities, soil types, geological layers and periods etc. and it is broadly used outside our discipline as well.

**Comment by ghwhitbread** [comment]
@nielsklazenga  along with many others (and why the original choice), but this class exists for the alignment of taxonomies, no?

**Comment by nielsklazenga** [comment]
@ghwhitbread yes, for the particular kind of taxonomies in which the elements are taxa.

**Comment by deepreef** [comment]
OK, well it's now more clear than ever that we all have different interpretations of these words. And if our little group has such a diversity of interpretations, then the spectrum of interpretations among the broader community is likely to be even wider.  (A common joke is that "a room with 10 taxonomists/informaticians will include at least 12 definitions of [taxon/concept/name/etc.]".)

Maybe it's best to avoid both "concept" and "circumscription".  

Again... are there any objections to simply sticking with `TaxonAlignment`? I agree that @ghwhitbread had it right from the start.

**Comment by nielsklazenga** [comment]
Let's do a poll. 

`TaxonAlignment` thumbs-up or thumbs-down (you'll have to explain in this case).

**Comment by jgerbracht** [comment]
TaxonAlignment is fine with me, again, as long as we unambiguously define what this means.   I could see this being applied to specific TNUs as well as to the underlying concepts

--
Jeff Gerbracht
Lead Application Developer
Birds of the World
Cornell Lab of Ornithology
607-254-2117

________________________________

**Comment by deepreef** [comment]
> I could see this being applied to specific TNUs as well as to the underlying concepts

As an aside, Dave Remsen, Nicolas Bailey and I have been meeting most Thursdays for an hour or two to think through the relationships of Protonyms, TNUs, and Taxon "Circumscriptions"/"Concepts"/Whatever-we-call-them.  Dave and I in particular have been discussing this for the past couple of decades.  We're very, very close to working out a model that we think solves it (in the sense of minting identifiers that represent meaningful circumscriptions, which can be algorithmically compared to other circumscriptions in terms of computing values of `TaxonAlignment` -- among other cool things).  Dave mentioned it at the Catalogue of Life Global Team meeting yesterday, as we're almost to the stage where we're ready to explain it to others and open it up to broader discussion (we're only just approaching the point where we're pretty sure we understand it ourselves).

As soon as we have some documents & diagrams & sample datasets to illustrate our ideas, I'll share with this group -- as I think it has a lot of potential for informing what we're trying to achieve here.  Watch this space...

**Comment by nielsklazenga** [comment]
Any further discussion on this is better had in #1  or the [TNC repo](/tdwg/tnc/issues), but, just to be clear about the terminology, the `TaxonConcept` we are talking about here is the TCS (1) Taxon Concept, which includes Taxonomic Name Usages (or overlaps, only including accepted TNUs) as well as the "shared" underlying concepts (cf. Avibase). It is not in the purview of the TCS 2 Task Group to change that.

**Comment by jliljeblad** [comment]
Thumbs up on `TaxonAlignment` from me. (Yeah, I'm lagging behind a bit)

**Comment by nielsklazenga** [comment]
I would like to make the case again to name this class `TaxonConceptAlignment`. I think leaving out the 'Concept' because of perceived issues with the word 'concept' is getting too hung up on words. These are alignments of TCS `TaxonConcept`'s, not Darwin Core `Taxon`'s.

**Comment by nielsklazenga** [comment]
There is a proposal, in issue [#225](https://github.com/tdwg/tcs2/issues/225#issuecomment-1644856311), to remove this class and have the [terms in the Taxon Relationship Type Vocabulary](https://github.com/tdwg/tcs2/issues?q=is%3Aissue+label%3A%22Taxon+Relation+Type+Vocabulary%22+) as properties of a Taxon Concept. I think the proposed change is significant enough to not just rely on comments in GitHub issues and take silence as agreement, so we are going to have a Task Group meeting about this in a few weeks' time.

**Comment by jliljeblad** [comment]
Yeah, I'm not comfortable enough with classes and such to understand the impact of this proposal.
